src/sentry/api/endpoints/organization_events_anomalies.py

```diff
@@ -0,0 +1,98 @@
+from drf_spectacular.utils import extend_schema
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization_events import OrganizationEventsV2EndpointBase
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers.base import serialize
+from sentry.apidocs.constants import (
+    RESPONSE_BAD_REQUEST,
+    RESPONSE_FORBIDDEN,
+    RESPONSE_NOT_FOUND,
+    RESPONSE_UNAUTHORIZED,
+)
+from sentry.apidocs.examples.organization_examples import OrganizationExamples
+from sentry.apidocs.parameters import GlobalParams
+from sentry.apidocs.utils import inline_sentry_response_serializer
+from sentry.models.organization import Organization
+from sentry.seer.anomaly_detection.get_historical_anomalies import (
+    get_historical_anomaly_data_from_seer_preview,
+)
+from sentry.seer.anomaly_detection.types import DetectAnomaliesResponse, TimeSeriesPoint
+
+
+@region_silo_endpoint
+class OrganizationEventsAnomaliesEndpoint(OrganizationEventsV2EndpointBase):
+    owner = ApiOwner.ALERTS_NOTIFICATIONS
+    publish_status = {
+        "POST": ApiPublishStatus.EXPERIMENTAL,
+    }
+
+    @extend_schema(
```

Comment:
in the future, if the api is experimental, documenting isn't explicitly required - 🥔 for doing this anyway!